### PR TITLE
Default text

### PR DESCRIPTION
**Pull request by njseeto**
Ensure the default text is editable and is only shown in preview mode when the user has customised it.

Story: https://trello.com/c/YSHgwzER/1382-placeholder-text

Co-authored-by: Brendan Butler <brendan.butler@digital.justice.gov.uk>
Co-authored-by: Tomas Destefi <tomas.destefi@digital.justice.gov.uk>

### CODE REVIEW

**Review by brenetic — APPROVED** [review]
🚀

File: app/controllers/preview_controller.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class PreviewController < ApplicationController
   layout 'metadata_presenter/application'
+
+  def service
+    @service ||= MetadataPresenter::Service.new(service_metadata, editor: editable?)
+  end
+  helper_method :service
```
Comment (tomas-stefano) [inline app/controllers/preview_controller.rb:7]: Maybe we can delete this line as there is no difference from the application controller

Comment (tomas-stefano) [reply app/controllers/preview_controller.rb:7]: I will see here locally

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**